params/config.go

```diff
@@ -121,16 +121,44 @@ type ChainConfig struct {
 	TxAllowListConfig               precompile.TxAllowListConfig               `json:"txAllowListConfig,omitempty"`               // Config for the tx allow list precompile
 }
 
+// FeeConfig specifies the parameters for the dynamic fee algorithm, which determines the gas limit, base fee, and block gas cost of blocks
+// on the network.
+//
+// The dynamic fee algorithm simply increases fees when the network is operating at a utilization level above the target and decreases fees
+// when the network is operating at a utilization level below the target.
 type FeeConfig struct {
-	GasLimit        *big.Int `json:"gasLimit,omitempty"`
-	TargetBlockRate uint64   `json:"targetBlockRate,omitempty"`
-
-	MinBaseFee               *big.Int `json:"minBaseFee,omitempty"`
-	TargetGas                *big.Int `json:"targetGas,omitempty"`
+	// GasLimit sets the max amount of gas consumed per block.
+	GasLimit *big.Int `json:"gasLimit,omitempty"`
+	// This sets the target rate of block production in seconds.
+	// A target of 2 will target producing a block every 2 seconds.
+	TargetBlockRate uint64 `json:"targetBlockRate,omitempty"`
+
```

Comment:
nit: space between these 2 and note between `GasLimit` and `TargetBlockRate`

Reply:
Added a new line between `GasLimit` and `TargetBlockRate`